### PR TITLE
Reboot on error

### PR DESCRIPTION
**Pull request by vladfi1**

### CODE REVIEW

File: gym_sts/communication/receiver.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ def __init__(self, fn, timeout: float = 50):
         self.timeout = timeout
         self.sleep_time = 0.05
         self.num_steps = int(timeout / self.sleep_time)
+        self.verbose = verbose
```
Comment (kronion) [inline gym_sts/communication/receiver.py]: Unused

File: gym_sts/envs/base.py [hunk 2]
```diff
@@ -391,11 +397,23 @@ def step(self, action_id: int) -> Tuple[dict, float, bool, dict]:
         try:
             obs = self.communicator._manual_command(action.to_command())
         except exceptions.StSError as e:
```
Comment (kronion) [inline gym_sts/envs/base.py]: This might be too specific, we probably ought to catch any `Exception`.

File: gym_sts/rl/mvp.py [hunk 3]
```diff
@@ -37,11 +37,17 @@ def check_rllib_bug(space: spaces.Space):
     headless=ff.Boolean(True),
     animate=ff.Boolean(False),
     build_image=ff.Boolean(False),
+    reboot_frequency=ff.Integer(20, "Reboot game every n resets."),
```
Comment (kronion) [inline gym_sts/rl/mvp.py]: This default is a little low, 50-100 is probably fine.

**Review by kronion — CHANGES_REQUESTED** [review]
One unused variable and a couple of suggestions, otherwise LGTM.

File: gym_sts/rl/mvp.py (hunk 3 above)
Comment (vladfi1) [reply gym_sts/rl/mvp.py]: Done.

File: gym_sts/envs/base.py (hunk 2 above)
Comment (vladfi1) [reply gym_sts/envs/base.py]: Done.

File: gym_sts/communication/receiver.py (hunk 1 above)
Comment (vladfi1) [reply gym_sts/communication/receiver.py]: Done.

**Pull request merged**